bigtable/admin.go

```diff
@@ -306,6 +338,8 @@ type TableInfo struct {
 	// DEPRECATED - This field is deprecated. Please use FamilyInfos instead.
 	Families    []string
 	FamilyInfos []FamilyInfo
+	// When true: deleting the table, the column families in the table and the instance containing the table is prohibited
```

Comment:
It's common style for Godoc to begin a comment on an exported field with the name of the field.